README.md

```diff
@@ -5,3 +5,36 @@ Welcome to the project maintainers' working group and committee. This group is i
 The first goal of this group will be to define their charter and present it to the board for approval. We encourage all project maintainers and contributors that have ideas on how the foundation can better support their projects to join and contribute.
 
 To join this group, please [create an issue](https://github.com/dotnet-foundation/wg-maintainers/issues/new/choose) in this repository and we will add you.
+
+## Monthly Meeting
+
+This committee meets at 19:00 UTC on the second Wednesday of every month. All community members are welcome to attend.
```

Comment:
I think we agreed on 21:00 UTC, did we not? ([meeting minutes](https://github.com/dotnet-foundation/wg-maintainers/pull/25/files#diff-b17eca63c54f62a42b80c6209e730bf691f2eee1cd2663cf7747a93d8eb33013R85) says 21:00 UTC)